internal/storage/pmem/pmem.go

```diff
@@ -36,6 +60,22 @@ func Mount(ctx context.Context, device uint32, target string) (err error) {
 		trace.Int64Attribute("device", int64(device)),
 		trace.StringAttribute("target", target))
 
+	source := fmt.Sprintf(pMemFmt, device)
+	return mountInternal(source, target)
+}
+
+// MountDM creates dm-linear block device and mounts it on `target`
+func MountDM(ctx context.Context, device uint32, deviceStart, deviceSize int64, target string) (err error) {
+	_, span := trace.StartSpan(ctx, "pmem::MountDM")
+	defer span.End()
+	defer func() { oc.SetSpanStatus(span, err) }()
+
+	span.AddAttributes(
+		trace.Int64Attribute("device", int64(device)),
+		trace.Int64Attribute("deviceStart", deviceStart),
+		trace.Int64Attribute("sectorSize", deviceSize),
+		trace.StringAttribute("target", target))
+
 	if err := osMkdirAll(target, 0700); err != nil {
```

Comment:
This is handled in the new `mountInternal` func right?

Reply:
oh lol, missed this one.